Source/EasyNetQ.Examples.DlxRetry/EntryPoint.cs

```diff
@@ -14,26 +14,28 @@
 );
 
 await bus.Advanced.QueueDeclareAsync(
-    "Events:Failed",
-    c => c.WithQueueType(QueueType.Quorum)
-        .WithDeadLetterExchange(Exchange.Default)
-        .WithDeadLetterRoutingKey("Events")
-        .WithMessageTtl(TimeSpan.FromSeconds(5)) // A fixed delay between retry attempts
-        .WithOverflowType(OverflowType.RejectPublish)
-        .WithDeadLetterStrategy(DeadLetterStrategy.AtLeastOnce)
+    queue: "Events:Failed",
+    arguments: new Dictionary<string, object>()
+        .WithQueueType(QueueType.Quorum)
+        .WithQueueDeadLetterExchange(Exchange.DefaultName)
+        .WithQueueDeadLetterRoutingKey("Events")
+        .WithQueueMessageTtl(TimeSpan.FromSeconds(5)) // A fixed delay between retry attempts
```

Comment:
do we really need this comment? is it possible to make it more obvious to remove this comment?

Reply:
It has been removed.